playbooks/robusta_playbooks/prometheus_enrichments.py

```diff
@@ -61,3 +64,32 @@ def prometheus_enricher(event: ExecutionBaseEvent, params: PrometheusQueryParams
     event.add_enrichment(
         [PrometheusBlock(data=prometheus_result, query=params.promql_query)],
     )
+
+
+def get_duplicate_kubelet_msg(rule_group) -> str:
+    """
+    checks if there are multiple kubelets and
+    """
+    NO_MESSAGE = ""
```

Comment:
Is it better to have a return value of Optional[str] and return None instead of NO_MESSAGE?